schema.graphql

```diff
@@ -13,26 +13,54 @@ enum PaymentType {
 
 type Service @entity {
   id: ID! # service id
-  createdAt: BigInt # timestamp of block creation
-  updatedAt: BigInt # timestamp of the last change
+  created: BigInt! # timestamp of block creation
+  updated: BigInt! # timestamp of the last change
   status: ServiceStatus! # service status
   buyer: User # service buyer
   seller: User # service seller
   sender: User # user that created the service (buyer or seller)
   recipient: User # other participating user (seller or buyer)
-  uri: String # metadata URI of service
   transaction: Transaction @derivedFrom(field: "service") # transaction associated with this service
   proposals: [Proposal!] @derivedFrom(field: "service") # proposals for this service
   platform: Platform # Platform on which service was created
+  cid: String
```

Comment:
Yes, if id is the cid and we need to remove the cid, then we need to store the cid to access the old description that we want to remove. Using service.description does not work since it's derived and loaded on query time.